### PR TITLE
Changecookingtime

### PR DESCRIPTION
**Pull request by hann992**
Kære Mads, vil du være sød at tjekke Patricks og min kode ud, og ser det godt ud, så godkende til Merge?
Mvh Andreas

### CODE REVIEW

File: Microwave.Test.Unit/CookControllerTest.cs [hunk 1]
```diff
@@ -35,6 +35,25 @@ public void StartCooking_ValidParameters_TimerStarted()
             timer.Received().Start(60);
         }
 
+        [Test]
+        public void StartCooking_ValidParameters_TimerUpdated()
```
Comment (Sylvaritius) [inline Microwave.Test.Unit/CookControllerTest.cs:39]: Test looks good.

File: Microwave.Test.Unit/CookControllerTest.cs [hunk 2]
```diff
@@ -35,6 +35,25 @@ public void StartCooking_ValidParameters_TimerStarted()
...
+        {
+            uut.UpdateTimer();
+
+            timer.Received().Update();
+        }
+
+        [Test]
+        public void StartCooking_ValidParameters_TimerRemaining()
```
Comment (Sylvaritius) [inline Microwave.Test.Unit/CookControllerTest.cs]: Test looks a-okay.

File: Microwave.Test.Unit/CookControllerTest.cs [hunk 3]
```diff
@@ -44,14 +44,11 @@ public void StartCooking_ValidParameters_TimerUpdated()
         }
 
         [Test]
-        public void StartCooking_ValidParameters_TimerRemaining()
+        public void UpdateTimer_ShowTime()
```
Comment (Sylvaritius) [inline Microwave.Test.Unit/CookControllerTest.cs:48]: Test changed is fine.

File: Microwave.Classes/Controllers/CookController.cs [hunk 4]
```diff
@@ -54,10 +54,14 @@ public void Stop()
         // Tilføjelsen: update timer.
         public void UpdateTimer()
         {
-            myTimer.Update();
+            if (isCooking)
```
Comment (Sylvaritius) [inline Microwave.Classes/Controllers/CookController.cs:57]: Godt at der er tilføjet tjek til edge case.

**Review by Sylvaritius — APPROVED** [review]
All the tests and changes to the code look great, and are hereby approved.
